homeassistant/components/thermopro/manifest.json

```diff
@@ -2,10 +2,18 @@
   "domain": "thermopro",
   "name": "ThermoPro",
   "bluetooth": [
+    {
+      "local_name": "TP358",
+      "connectable": true
+    },
     {
       "local_name": "TP35*",
       "connectable": false
     },
+    {
+      "local_name": "TP393",
+      "connectable": true
+    },
```

Comment:
This isn't needed since there is a wildcard match below

Reply:
Don't/Shouldn't I set connectable=true for these two devices as I later connect to them to set the datetime?

Reply:
Everything else will still work without being connectable so we wouldn't want to prevent users who don't have a Bluetooth adapter that can make connections (ie Shelly) from being able to use these devices